controllers/applicationvolumereplication_controller.go

```diff
@@ -62,18 +62,19 @@ const (
 	MWTypePV string = "pv"
 )
 
-type S3StoreInterface interface {
-	DownloadPVs(ctx context.Context, r client.Reader,
-		s3Endpoint string, s3SecretName types.NamespacedName,
+type pvDownloader interface {
```

Comment:
I am fine changing the name, but why did you make it private?

Reply:
I was trying to minimize the set of things that needed to be made public. Call it my experiment with Golang.